example/gaussian_blur.cpp

```diff
@@ -0,0 +1,18 @@
+#include <boost/gil/extension/io/jpeg.hpp>
```

Comment:
Licence and copyright is missing

As I [mentioned on Slack](https://cpplang.slack.com/archives/CSVT0STV2/p1583593764075500), please try to apply the initial clean-up according to the GIL guidelines as well as Boost general guidelines, links on right hand side here https://www.boost.org/development/index.html